src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -294,6 +294,7 @@ impl<'a, 'tcx> LinkCollector<'a, 'tcx> {
         module_id: DefId,
         item_name: Symbol,
         item_str: &'path str,
+        span: rustc_span::Span,
```

Comment:
nit: I would import this as just `Span`.

Reply:
Done. I also consistently made `span` the first parameter.